graph-db/scripts/deploy-bamboo.sh

```diff
@@ -12,7 +12,7 @@ node_modules
 EOF
 
 cat <<EOF > Dockerfile
-FROM node:16
+FROM node:18.16-bullseye
```

Comment:
This is what `cmr-graphl` is currently set to and I saw a chat in `#ci/cd` about needing this, the dependencies may be updated enough at this point that we could use `node:18-bullseye`